source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.h

```diff
@@ -25,6 +25,12 @@ class SimpleHttpCache : public HttpCache, public Singleton::Instance {
     Http::ResponseTrailerMapPtr trailers_;
   };
 
+  // Returns a Key with the vary header added to custom_fields.
+  // It is an error to call this with headers that don't include vary.
+  // Returns nullopt if the vary headers are no longer valid.
```

Comment:
Can you say what "no longer valid" means?

Reply:
Updated to
```
// Returns nullopt if the vary headers in the response are not
// compatible with the VaryAllowList in the LookupRequest.
```